ddtrace/commands/ddtrace_run.py

```diff
@@ -107,11 +107,10 @@ def main():
     if os.path.basename(executable) == "uwsgi":
         print(
             (
-                "ddtrace-run is not supported with uwsgi. "
+                "ddtrace-run is no longer supported with uwsgi since 0.48.0 release. "
```

Comment:
Do we want to be more aggressive with the wording?

> ddtrace-run support with uwsgi is deprecated and will be disallowed in version 0.49.0
> See <link> for details.

?

Reply:
Better?

```
DeprecationWarning: ddtrace-run is no longer supported with uWSGI since 0.48.0 release. Remove ddtrace-run and update your uWSGI configuration following https://ddtrace.readthedocs.io/en/stable/advanced_usage.html#uwsgi.
```

Reply:
I think what this doesn't convey is that today is is deprecated/suggested you move to something else, tomorrow it is going to be rejected and stop working.

Reply:
Yeah I'm not sure if we want to call it a deprecation or not. It seems like we have plans to fix it so it'd be a weird experience as a user to have something you're using be deprecated and then added back a few releases later.

I'd be more inclined to warn that we know of potential issues with `ddtrace-run` and let the user continue at their own risk